js/adapt-xapi.js

```diff
@@ -78,6 +80,25 @@ define(function(require) {
       this.listenTo(Adapt, "xapi:stateLoaded", this.restoreState);
     },
 
+    onComponentComplete: function(component) {
```

Comment:
The onComponentComplete and onAssessmentComplete functions are the same apart from the new ComponentStatemetnModel vs new AssessmentStatementModel - could we combine them and create a StatementModelFactory (see my Java experience slipping in!)